korgan/kaspi_ofd.py

```
@@ -0,0 +1,393 @@
+    payment_method = _value_after_label(lines, ("Оплачено", "Способ оплаты"))
+
+    if not sale_datetime:
+        sale_datetime = query.get("sale_date") or query.get("t", "")
+    if not rnm and parsed_url.path == _KASPI_FISCAL_PATH:
+        rnm = query.get("f", "")
+    if not fp and parsed_url.path == _KASPI_FISCAL_PATH:
+        fp = query.get("i", "")
+
+    amount_kzt = _query_amount(query)
+    if not amount_kzt:
+        amount_raw = _value_after_label(lines, ("Итого", "К оплате", "Сумма оплаты", "Сумма покупки", "Сумма"))
```

Ruff (0.16.2)

[warning] String contains ambiguous `К` (CYRILLIC CAPITAL LETTER KA). Did you mean `K` (LATIN CAPITAL LETTER K)?

(RUF001)